### PR TITLE
added php-fpm options to install documentation

### PR DESCRIPTION
**Pull request by kapralVV**
@iklymenko 
Please let me know if some additional options/configs should be added.

### CODE REVIEW

**Comment by orocla** [comment]
Thank you for your pull request.

It looks like this may be your first contribution to an Oro, Inc. open source project, in which case you'll need to sign a Contributor License Agreement (CLA) at http://www.orocrm.com/contributor-license-agreement/

If you've already signed the CLA, it's possible we don't have your GitHub username or you're using a different email address. GitHub uses the email address you set in your local Git configuration to associate commits with your GitHub account. Please sign the CLA again using the correct GitHub username and email address or see this help article on [setting the email on your git commits](https://help.github.com/articles/setting-your-email-in-git/).

Once you've signed the CLA, please allow for some time for the submission to be processed.

**Comment by iklymenko** [comment]
@kapralVV, please fix issue found by the CI. Travis builds should be green.

File: book/installation.rst [hunk 1]
```diff
@@ -267,17 +269,35 @@ Step 3. Configure the Webserver
...
+   pm = dynamic
+   pm.max_children = 128
+   pm.start_servers = 8
+   pm.min_spare_servers = 4
+   pm.max_spare_servers = 8
+   pm.max_requests = 512
+
+   catch_workers_output = yes
+
+.. note:: Make sure that options fastcgi_pass for nginx and listen for php-fpm are configured accordingly
```
Comment (iklymenko) [inline book/installation.rst]: @kapralVV please take option names in quotes, thus you'll eliminate notional confusion with word 'listen'

File: book/installation.rst [hunk 2]
```diff
@@ -267,17 +269,35 @@ Step 3. Configure the Webserver
...
-
 .. caution::
 
     Make sure that the web server user has permissions for the "log" directories of the application. 
     
     More details on the file permissions configuration are available in the official Symfony 
     documentation of *"`Setting up Permissions`_"* 
 
+
+**Configuration PHP-FPM**,
```
Comment (iklymenko) [inline book/installation.rst]: @kapralVV replace with "PHP-FPM Configuration" and remove the comma

**Comment by iklymenko** [comment]
Do we have any other settings, instead of PHP, set for the demo instance that should be shared?

**Comment by kapralVV** [comment]
Thanks for such good remarks.
Fixed. The config file for php-opcache was added as well.
Please re-view

File: book/installation.rst [hunk 3]
```diff
@@ -219,8 +219,8 @@ Create an empty database, such that its values correspond to the
 
     CREATE EXTENSION "uuid-ossp";
 
-Step 3. Configure the Webserver
--------------------------------
+Step 3. Web-Server Configuration
```
Comment (DimaSoroka) [inline book/installation.rst]: Let use "Web Server" instead "Web-Server"

**Pull request merged**